src/Engine/Renderer/Material/BlinnPhongMaterial.cpp

```diff
@@ -101,7 +99,7 @@ bool BlinnPhongMaterial::isTransparent() const {
 
 void BlinnPhongMaterial::registerMaterial() {
     // For resources access (glsl files) in a filesystem
-    std::string resourcesRootDir = {Core::Resources::getRadiumResourcesDir()};
+    auto resourcesRootDir {RadiumEngine::getInstance()->getRadiumResourcesDir()};
```

Comment:
Simpler to write if the `getRadiumResourcesDir` is static : 
```suggestion
    auto resourcesRootDir {RadiumEngine::getRadiumResourcesDir()};
```